python/lsst/ts/externalscripts/auxtel/latiss_acquire_and_take_sequence.py

```diff
@@ -177,6 +188,41 @@ def get_schema(cls):
         """
         return yaml.safe_load(schema_yaml)
 
+    def format_as_array(self, value, recurrences):
```

Comment:
This seems like a general purpose utility method. Consider moving it to https://github.com/lsst-ts/ts_standardscripts/blob/develop/python/lsst/ts/standardscripts/utils.py